README.md

```diff
@@ -3,59 +3,72 @@
 
 ## go-sdk
 
-A Go SDK to make it easy to talk to the [Transloadit](https://transloadit.com) REST API.
+A GO Integration for [Transloadit](https://transloadit.com)'s file uploading and encoding service
 
-We also offer [transloadify](https://github.com/transloadit/transloadify) that bundles much of the technology inside this SDK as a commandline utility.
+## Intro
 
-## Example
+[Transloadit](https://transloadit.com) is a service that helps you handle file uploads, resize, crop and watermark your images, make GIFs, transcode your videos, extract thumbnails, generate audio waveforms, and so much more. In short, [Transloadit](https://transloadit.com) is the Swiss Army Knife for your files.
 
-```go
-// Create client
-options := transloadit.DefaultConfig
-options.AuthKey = "TRANSLOADIT_KEY"
-options.AuthSecret = "TRANSLOADIT_SECRET"
-client, err := transloadit.NewClient(options)
-if err != nil {
-    panic(err)
-}
+This is a GO SDK to make it easy to talk to the [Transloadit](https://transloadit.com) REST API.
 
-// Initialize new assembly
-assembly := client.CreateAssembly()
-
-// Add input file to upload
-assembly.AddFile("myimage", "./lol_cat.jpg")
-
-// Add instructions, e.g. resize image to 75x75px
-assembly.AddStep("resize", map[string]interface{}{
-    "robot":           "/image/resize",
-    "width":           75,
-    "height":          75,
-    "resize_strategy": "pad",
-    "background":      "#000000",
-})
-
-// Wait until transloadit is done processing all uploads
-// and is ready to download the results
-assembly.Blocking = true
-
-// Start the upload
-info, err := assembly.Upload()
-if err != nil {
-    panic(err)
-}
+## Install
 
-fmt.Printf("You can view the result at: %s\n", info.Results["resize"][0].Url)
+```bash
+go get github.com/transloadit/go-sdk
 ```
 
-For examples on how to use templates, non-blocking processing and more, take a look at [`examples/`](https://github.com/transloadit/go-sdk/tree/master/examples).
+The Go SDK requires Go 1.1 or higher.
 
-## Installation
+## Usage
 
-```bash
-go get github.com/transloadit/go-sdk
+```go
+import (
+  "fmt"
+  "github.com/transloadit/go-sdk"
+)
+
+func main () {
+  // Create client
```

Comment:
Please adjust the indention to have a consistent style in the full example. Go uses tabs as indentation which is usually displayed as four spaces.